### PR TITLE
[release-4.16] OCPBUGS-44301: Use copy() to duplicate a net.IP

### PR DESCRIPTION
**Pull request by pliurh**
Fix regression in 1d1d0ba19a4ff2cd4f04710213e363d3f904c39c

### CODE REVIEW

File: pkg/network/node/networkpolicy.go [hunk 1]
```diff
@@ -822,7 +823,8 @@ func (np *networkPolicyPlugin) parsePeerFlows(npns *npNamespace, npp *npPolicy,
 					sel, _ := metav1.LabelSelectorAsSelector(peer.NamespaceSelector)
 					if _, ok := np.nsMatchCache[sel.String()].matches[HostNetworkNamespace]; ok {
 						for _, network := range np.node.networkInfo.ClusterNetworks {
-							cidrIP := network.ClusterCIDR.IP
+							cidrIP := make(net.IP, len(network.ClusterCIDR.IP))
+							copy(cidrIP, network.ClusterCIDR.IP)
```
Comment (danwinship) [inline pkg/network/node/networkpolicy.go:827]: hm... I feel like there ought to be an easier way to do this, but I can't think of one really...

**Comment by zhaozhanqi** [comment]
/hold
for pre-merge testing

**Comment by zhaozhanqi** [comment]
pre-merge tested this patch and looks good. 
/hold cancel
/qe-approved

**Pull request merged**